content/blog/lift-and-transform-apps-with-hpe-csi-driver-for-kubernetes.md

```diff
@@ -0,0 +1,283 @@
+
+Also, remember to have backups or volume replicas and a contingency plan for any unplanned event.
+
+Once the VM has been deleted from vSphere, the remnants on the storage array can be seen as offline volumes. The metadata volumes have been permanently removed.
+
+![Offline HPE Storage Nimble volume](/img/offline-screen-shot-2022-06-14-at-5.45.27-pm.png "Offline HPE Storage Nimble volume")
+
+An important detail to understand is that `importVolumeName` only works for offline volumes on HPE Alletra 6000 and Nimble Storage.
+
+Ok, let’s import the database into its final state.
+
+```markdown
+$ helm install my-prod bitnami/mariadb -f values-import.yaml
+```
+
+Once the database is up, we can connect to the database and verify the contents yet again.
+
+```
+$ kubectl exec -it my-prod-mariadb-0 -- bash
+I have no name!@my-prod-mariadb-0:/$ mysql -u mmattsson -ppassword < /bitnami/mariadb/scripts/test_employees_sha.sql
+INFO
+TESTING INSTALLATION
+computation_time
+00:00:48
+summary	result
+CRC	OK
+count	OK
+```
+
+Back on the storage array, the volume has automatically been renamed and now corresponds to the `PersistentVolume` name known to Kubernetes.
+
+![PersistentVolume hosted on HPE Nimble Storage](/img/pvc-screen-shot-2022-06-14-at-6.39.40-pm.png "PersistentVolume hosted on HPE Nimble Storage")
+
+It’s also safe to move the volume out of the vVol datastore folder, it’s a non-disruptive operation and may be carried out by a storage administrator.
+
+# Summary
+
+While this blog only skims the surface of possibilities of performing IT transformations from thirty-year-old paradigms, it gives an idea of what tools DevOps teams have at their disposal while taking a crack at one of the biggest challenges in application modernization. The ability to iterate safely without production disruption is key. Quite frankly, I performed a dozen clone imports before I got every parameter right before the final import while writing this blog post. There is no easy button, you must put in the work! 
+
+I’m gearing up for [HPE Discover 2022](http://hpe.com/discover). If you’d like to connect and talk shop, I’ll be in the HPE Developer Community Hack Shack on the main expo floor. [We have treasure hunts, hacking challenges and meetups](https://developer.hpe.com/blog/don%E2%80%99t-miss-all-things-software-in-the-hack-shack-at-hpe-discover/). Find my session “Get Started with Persistent Storage for Kubernetes with the HPE CSI Driver” in the HPE Discover [content catalog](https://content.attend.hpe.com/go/agendabuilder.sessions/?l=1049&sid=24991_10161&locale=en_US).
+
+![Get Started with Persistent Storage for Kubernetes with the HPE CSI Driver](/img/hpe-dlv-2022-csi-hackshack-revc.png "Get Started with Persistent Storage for Kubernetes with the HPE CSI Driver")
+
+The team is also on the HPE Developer Community Slack, sign up and login to join the conversation on everything HPE Developer related. See you there!
```

Comment:
Suggest to hotlink the HPE Developer Community Slack (https://slack.hpedev.io/):

The team is also on the [HPE Developer Community Slack](https://slack.hpedev.io/), sign up and login to join....